src/FlashMessage/FlashManager.php

```diff
@@ -103,6 +112,26 @@ public function getHandledTypes(): array
         return array_keys($this->getTypes());
     }
 
+    public function getCssClass(string $type, string $default = ''): string
```

Comment:
IMO, regardless the "class" HTML attribute is commonly used by CSS selectors, the fact involving the term "CSS" in this class is out of its scope. I know the existing properties are already using this term, but I think we could find a better name.

Reply:
getFlashmessageClass?

Reply:
I'd prefer something that make clear the fact it is used with presentation / rendering purposes (`getRenderedClassAttribute()`, or even more verbose `getRenderedHtmlClassAttribute()`).
We could also make a good description for this method (or even the interface), explaining how is it tied to the view layer.